### DISCUSSION TITLE
Question about Software architecture

### DISCUSSION THREAD

**Discussion by timo1302** [mulder82/immich-slideshow#5] [Q&A]
Hello Immich-Slideshow Community,

i want to run this software at my raspberry Pi 4 (ARM). Is the software with this compatible?

Regards
Timo

**Comment by mulder82** [comment]
Hi, the only requirements for the ImmichSlideshow add-on is a working web browser and an immich server running. On the raspberry PI4 both should work without a problem. The add-on itself is a 100% java script executed in the client's browser.
